packages/eslint-plugin/README.md

```
@@ -0,0 +1,51 @@
+
+## Rules
+
+| Rule | Default | Autofix | What it catches |
+|---|---|---|---|
+| `defer-on-module-script` | warn | ✓ | `<script type="module" defer>` (defer is redundant) |
+| `empty-meta-content` | warn |   | `<meta name="description" content="">` |
+| `no-deprecated-props` | error | ✓ | v2 props: `children`, `hid`, `vmid`, `body: true` |
+| `no-html-in-title` | warn |   | HTML chars in `title` (will be escaped, not rendered) |
+| `no-unknown-meta` | warn | ✓ | typos in `name` / `property` (Levenshtein-suggested fix) |
+| `non-absolute-canonical` | warn |   | relative URLs in `<link rel="canonical">` |
+| `numeric-tag-priority` | warn | suggestion | numeric `tagPriority` (suggests `'critical' | 'high' | 'low'`) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)